src/ImageSharp/Formats/Tiff/Writers/TiffBiColorWriter.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Buffers;
+
+using SixLabors.ImageSharp.Formats.Experimental.Tiff.Compression;
+using SixLabors.ImageSharp.Memory;
+using SixLabors.ImageSharp.Metadata;
+using SixLabors.ImageSharp.PixelFormats;
+using SixLabors.ImageSharp.Processing;
+
+namespace SixLabors.ImageSharp.Formats.Experimental.Tiff.Writers
+{
+    internal class TiffBiColorWriter<TPixel> : TiffBaseColorWriter<TPixel>
```

Comment:
Any concrete implementations should be sealed.

Reply:
done with b0e965fd